src/metrics/metrics.cc

```diff
@@ -34,8 +34,25 @@
 #endif
 
 #include "metrics.h"
+#include "metrics/master.h"
 #include "slogger/slogger.h"
 
+// Quick summary on the current implementation from 17/07/24
```

Comment:
I think in the final version, there should not be implementation details referencing specific dates.

Reply:
My worry is that comments go out of date fast, but I guess that one can look at git blame instead.